src/main/java/com/revature/maadcars/controllers/VehicleController.java

```diff
@@ -128,4 +139,27 @@ public VehicleController(VehicleService vehicleService){
         logger.trace("Successfully deleted vehicle with id: " + vehicle_id);
         return ResponseEntity.ok().build();
     }
+
+    /**
+     * Maps "PUT vehicles/transfer/{vehicle_id}/to/{new_user_id}"
```

Comment:
Document what this method does with this mapping.

Reply:
Updated, should be there now.